com.bitwarden.desktop.yaml

```diff
@@ -1,47 +1,151 @@
 app-id: com.bitwarden.desktop
+branch: stable
 runtime: org.freedesktop.Platform
 runtime-version: '20.08'
 sdk: org.freedesktop.Sdk
 base: org.electronjs.Electron2.BaseApp
 base-version: '20.08'
-command: bitwarden
 separate-locales: false
+command: start-bitwarden
+sdk-extensions:
+  # https://aur.archlinux.org/packages/bitwarden/?O=20&PP=10#comment-753128
+  - org.freedesktop.Sdk.Extension.node12
 finish-args:
+  - --device=dri
   - --share=ipc
+  # TODO: add wayland socket when electron supports wayland
+  # https://github.com/electron/electron/issues/10915
```

Comment:
Looks like this issue has been closed and Electron (~)supports Wayland since V12. Since Bitwarden is using V19 currently is it worth adding this now?